bin/console

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env ruby
+
+require 'bundler/setup'
+require 'rdoc'
+
+require_relative '../doc/rdoc/markup_reference'
```

Comment:
Would you be able to add a comment to explain why this is needed for future reference.

Reply:
Done!